### PR TITLE
Added setting and separate style sheet for MIT login split

### PR DESCRIPTION
**Pull request by jamieandyou**
Needed to make sure the new MIT login screen split doesn't affect other online campuses.

So added a setting and separate style sheet for MIT login split

### CODE REVIEW

File: config.php [hunk 1]
```diff
@@ -46,6 +46,9 @@
 $THEME->sheets[] = 'mobile';
 $THEME->sheets[] = 'mobilemenu';
 $THEME->sheets[] = 'flags16';
+if ((!empty($THEME->settings->mit_courses_split)) && ($THEME->settings->mit_courses_split == '1')) {
```
Comment (davidjnorth) [inline config.php]: If it's equal to 1 then won't it be not empty by default? So you'll only need the last condition. And the extra brackets around each condition isn't necessary.

File: settings.php [hunk 2]
```diff
@@ -109,6 +109,15 @@
     $setting->set_updatedcallback('theme_reset_all_caches');
     $temp->add($setting);
 
+    // Login MIT courses split.
+    $name = 'theme_universal/mit_courses_split';
```
Comment (davidjnorth) [inline settings.php]: Can we change this to not have underscores. Moodle coding style for variables is to not have underscores https://docs.moodle.org/dev/Coding_style

Comment (jamieandyou) [inline settings.php]: Cool agreed

File: config.php (hunk 1 above)
Comment (jamieandyou) [inline config.php]: Agreed

**Comment by davidjnorth** [comment]
Nice 👍

**Pull request merged**